pydm/widgets/image.py

```diff
@@ -1,5 +1,6 @@
 from qtpy.QtWidgets import QActionGroup
-from qtpy.QtCore import Signal, Slot, Property, QTimer, Q_ENUMS, QThread
+from qtpy.QtCore import Signal, Slot, Property, QTimer, QThread
+from PyQt5.QtCore import Q_ENUM
```

Comment:
does this work for pyside6?

Reply:
Also would it make it so anyone using pyside6 have to also have PyQt5 installed as well?

Reply:
Wait I read your the previous PR and see that a pyside6 enum will be added in the future